src/styles/main.scss

```diff
@@ -1,3 +1,243 @@
+$color-dark: #060b35;
+$color-small: #616070;
+$color-button: #00acdc;
+$color-bck: #f3f3f3;
+$width-card: 200px;
+$card-margin-right: 48px;
+$padding-horizont: 41px;
+$padding-vertical: 51px;
+$card-margin-bottom: 46px;
+$width: ($width-card * 4) + ($card-margin-right * 3) + (2 * $padding-horizont);
+$head-padding: 0 50px;
+$link-rightmarg: 20px;
+$link-padding: 23px 0;
+$font-ten: 10px;
+$font-twelve: 12px;
```

Comment:
split your styles code to separate files:

bem blocks, scss features like variables, mixins and so on, reset file